runtime/src/committee.rs

```diff
@@ -481,6 +484,36 @@ mod tests {
         }
     }
 
+    pub struct TestChangeMembers;
+    impl ChangeMembers<IdentityId> for TestChangeMembers {
```

Comment:
Suggestion: ChangeMembers, InitializeMembers implemented for `()` in `srml_support`. So, if we don't need to test this behavior, perhaps we can just assign  `type MembershipInitialized = ();` ... keeps things simpler.  Same for `type MembershipChanged = TestChangeMembers;`